src/main/java/com/kodilla/ecommercee/domain/dto/ProductDto.java

```diff
@@ -0,0 +1,17 @@
+package com.kodilla.ecommercee.domain.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+import java.math.BigDecimal;
+
+@Getter
+@AllArgsConstructor
+public class ProductDto {
+
+    private Long id;
+    private String name;
+    private String description;
+    private BigDecimal price;
+    private int groupId;
```

Comment:
Shouldn't groupId be Long type?

Reply:
You're right, I didn't notice that. I'll change it, wait for other comments and fix it with one commit